src/test/java/facades/WeatherFacadeTest.java

```diff
@@ -114,4 +117,38 @@ public void testGetRightDate() throws NotFoundException {
         assertTrue(res.contains(weatherTwo));
     }
 
+    // https://docs.oracle.com/javase/7/docs/api/java/util/Calendar.html
+    // https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html
+    @Test
+    public void testGet5DaysWeather() throws Exception {
+        Date rightNow = Calendar.getInstance().getTime();
+        SimpleDateFormat y = new SimpleDateFormat("yyyy");
+        String year = y.format(rightNow);
+        SimpleDateFormat m = new SimpleDateFormat("MM");
+        String month = m.format(rightNow);
+        SimpleDateFormat d = new SimpleDateFormat("dd");
+        String day = d.format(rightNow);
```

Comment:
The endpoint changes the date every day. 

No way to predict the weather, but we know what day it is, so that is the only thing we can test on.